### PR TITLE
Feature/#1 basic setup

### PR DESCRIPTION
**Pull request by ceichinger**
- Folder Structure
- Custom font
- Prepare i18n (Internationalization) with i18next
- Redux
- React navigation with basic Intro and Tabbar Stack
- Empty Screens

### CODE REVIEW

File: App.tsx [hunk 1]
```diff
@@ -1,19 +1,51 @@
...
+import { createStackNavigator } from '@react-navigation/stack';
+import * as React from 'react';
+import { StyleSheet, YellowBox } from 'react-native';
+import { AppearanceProvider } from 'react-native-appearance';
+import { enableScreens } from 'react-native-screens';
+import { SafeAreaProvider } from 'react-native-safe-area-context';
+import { Provider, connect } from 'react-redux';
+// @ts-ignore
+import { PersistGate } from 'redux-persist/integration/react';
+// import QuickActions from 'react-native-quick-actions';
```
Comment (seb0zz) [inline App.tsx:13]: Do you want to keep this comment or can we drop it?

File: App.tsx [hunk 2]
```diff
@@ -1,19 +1,51 @@
...
-  container: {
-    flex: 1,
-    backgroundColor: '#fff',
-    alignItems: 'center',
-    justifyContent: 'center',
-  },
-});
+  if (!isLoadingComplete) {
+    return null;
+  } else {
```
Comment (seb0zz) [inline App.tsx:38]: The `else` can be removed here

File: src/navigation/AppNavigator.tsx [hunk 3]
```diff
@@ -0,0 +1,106 @@
+import React, { useRef, useState, useEffect } from 'react';
+import { NavigationContainer } from '@react-navigation/native';
+import { createStackNavigator } from '@react-navigation/stack';
+import { NativeModules } from 'react-native';
+import Constants from 'expo-constants';
+import * as Localization from 'expo-localization';
+
+import currentVersion from '../config/legal.json';
+import LinkingConfiguration from './LinkingConfiguration';
+// import { connectToStore, addPurchaseListener, disconnectFromStore } from '../utils/payment'
```
Comment (seb0zz) [inline src/navigation/AppNavigator.tsx:10]: Same with these comments?

File: App.tsx (hunk 1 above)
Comment (ceichinger) [reply App.tsx:13]: I want to keep it. We will use it later ✌️

**Pull request merged**